Core/scripts/dirac-externals-requirements.py

```diff
@@ -69,14 +73,11 @@ def setInstallType( val ):
 
 gLogger.notice( "Requesting installation of %s" % ", ".join( [ "%s%s" % ( reqName, reqDict[ reqName ][0] ) for reqName in reqDict ] ) )
 
-from pip.index import PackageFinder
-from pip.req import InstallRequirement, RequirementSet
-from pip.locations import build_prefix, src_prefix
-
 requirement_set = RequirementSet(
-    build_dir = build_prefix,
+    build_dir = user_dir,
```

Comment:
This script is used in the special case when /RequiredExternals/ is found in releases.cfg. To be honest, I don't know by who (not by LHCb) and IIUC it's for some user installation, probably by pilots. It's an alternative way to have a package on the worker node, where only the user space is usable.